test/langtools/tools/javac/processing/model/element/TestExecutableReceiverType.java

```diff
@@ -45,8 +46,14 @@ public boolean process(Set<? extends TypeElement> annotations,
                            RoundEnvironment roundEnv) {
         if (!roundEnv.processingOver()) {
             int count = 0;
-            count += testType(elements.getTypeElement("MethodHost"));
-            count += testType(elements.getTypeElement("MethodHost.Nested"));
+            for (ExecutableElement e : ElementFilter.methodsIn(
+                  roundEnv.getElementsAnnotatedWith(ReceiverTypeKind.class))) {
+              count += testExecutable(e);
```

Comment:
nit: indentation should be 4 spaces here, same issue below

Reply:
Done